### DISCUSSION TITLE
can't sign in in my github mobile app android

### DISCUSSION THREAD

**Discussion by LucasHillebrand** [community/community#65472] [Mobile] [answered]
### Select Topic Area

Question

### Feature Area

Projects

### Body

i can sign in in the browser that opens when signing in, but it doesn't appear to work on the app itself
![err](https://github.com/community/community/assets/71591567/c1a18268-7dbf-4de8-bea2-d43bb38cc1ca)

### Guidelines

- [x] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @LucasHillebrand ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by kangarruu** [comment, marked as answer]
@LucasHillebrand, we're sorry you're experiencing this issue, thanks for reaching out! It looks like you're using Brave browser when signing in. Brave has recently made changes that cause web redirects to external apps to be turned off by default. You would need to manually enable "Allow app links to open outside of Brave" in your Brave Shields & Privacy settings to redirect back to the GitHub Mobile app. Would you mind checking your settings and seeing if this resolves the issue for you?

**Reply by Rex-82** [reply]
I had the same issue and this solved it. Thank you!
